src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -1208,17 +1208,17 @@ private BoundExpression BindSizeOf(SizeOfExpressionSyntax node, DiagnosticBag di
         }
 
         /// <returns>true if managed type-related errors were found, otherwise false.</returns>
-        private static bool CheckManagedAddr(TypeSymbol type, SyntaxNode node, DiagnosticBag diagnostics)
+        internal static bool CheckManagedAddr(TypeSymbol type, Location location, DiagnosticBag diagnostics)
         {
             var managedKind = type.ManagedKind;
             if (managedKind == ManagedKind.Managed)
             {
-                diagnostics.Add(ErrorCode.ERR_ManagedAddr, node.Location, type);
+                diagnostics.Add(ErrorCode.ERR_ManagedAddr, location, type);
                 return true;
             }
             else if (managedKind == ManagedKind.UnmanagedWithGenerics)
             {
-                var supported = CheckFeatureAvailability(node, MessageID.IDS_FeatureUnmanagedConstructedTypes, diagnostics);
+                var supported = CheckFeatureAvailability(MessageID.IDS_FeatureUnmanagedConstructedTypes, diagnostics, location);
```

Comment:
This is gonna conflict with my PR where I remove the ability to check for feature availability using a location: https://github.com/dotnet/roslyn/pull/37513

Reply:
Ok, how would you like to proceed? I can merge master into this after 37513 gets merged and handle it if you’d like.

Reply:
I merged master in, but since master hasn't flowed to 16.4-preview1 yet, the fix from 37513 will continue to show in here for a little while.